metrics/statsd.go

```diff
@@ -29,35 +31,41 @@ func NewStatsdMonitor(client statsdClient, prefix string) *StatsdMonitor {
 }
 
 // RecordRun records a run behavior
-func (m *StatsdMonitor) RecordRun(recipe recipe.Recipe, duration int, success bool) {
+func (m *StatsdMonitor) RecordRun(run agent.Run) {
 	m.client.Timing(
-		m.createMetricName(runDurationMetricName, recipe, success),
-		int64(duration),
+		m.createMetricName(runDurationMetricName, run.Recipe, run.Success, run.RecordCount),
+		int64(run.DurationInMs),
 	)
 	m.client.Increment(
-		m.createMetricName(runMetricName, recipe, success),
+		m.createMetricName(runMetricName, run.Recipe, run.Success, run.RecordCount),
+	)
+	m.client.IncrementByValue(
```

Comment:
i'm still thinking if we need another metric type for data count.
at first im thinking data count is just a metadata in a metric so we can put it as a tag.

let me try it out first if it we can actually get something out of this new metric.